strings/scripts/pallet_town/player_room.string

```diff
@@ -0,0 +1,11 @@
+#org @gText_PlayerRoom_Rival_WakeUp
+[GREEN][PLAYER]...\p[PLAYER]...\pWake up, [PLAYER]...!!
+
+#org @gText_PlayerRoom_Rival_Late
+[GREEN][RIVAL]: Come on come on!\p Or we're gonna be late!
+
+#org @gText_PlayerRoom_Player_Festival1
+[RED][PLAYER]: Late...?\pOh, right\pThe Millennial Art Festival is today.
```

Comment:
Like the Green for Rival but Fire Red uses red text for female characters unless we change that

Reply:
@Stevebel Changed to orange until we figure out somethin better